bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,46 @@
+import aiohttp
+import discord
+from discord.ext import commands
+
+from bot.constants import Tokens
+
+UnClient_id = Tokens.unsplash_key
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.current_channel = None
```

Comment:
Is this used for anything?

Reply:
you mean self.current_channel? I got rid of it i thought